PurchasesTests/UnitTestsConfiguration.storekit

```diff
@@ -0,0 +1,93 @@
+{
+  "identifier" : "B495CDA2",
```

Comment:
for now, this is just like the old `Configuration.storekit` file, but we can update it with more stuff as needed. And we can have multiple of these as well.